src/devices/Tca955x/samples/README.md

```diff
@@ -0,0 +1,33 @@
+﻿# Usage of the Tca9554 or Tca9555
+
+## Example 1
+
+Use the Tca9554 or Tca9555 class directly and write to the specific register.
+First, write to the configuration register to define either input or output (a high bit stands for an input).
+Read inputs with the input register or write outputs with the output register.
+```csharp
+I2cConnectionSettings i2cConnectionSettings = new(1, 0x20);
```

Comment:
rather than 0x20, you should use this new constant introduced, it's going to be cleaner